BraintreeDropIn.podspec

```diff
@@ -27,11 +27,11 @@ Pod::Spec.new do |s|
     s.source_files  = "Sources/BraintreeDropIn/**/*.{h,m}"
     s.public_header_files = "Sources/BraintreeDropIn/Public/BraintreeDropIn/*.h"
     s.frameworks = "UIKit"
-    s.dependency "Braintree/Card", "~> 4.32"
-    s.dependency "Braintree/Core", "~> 4.32"
-    s.dependency "Braintree/UnionPay", "~> 4.32"
-    s.dependency "Braintree/PaymentFlow", "~> 4.32"
-    s.dependency "Braintree/PayPal", "~> 4.32"
+    s.dependency "Braintree/Card", "~> 5.0.0-beta"
+    s.dependency "Braintree/Core", "~> 5.0.0-beta"
+    s.dependency "Braintree/UnionPay", "~> 5.0.0-beta"
+    s.dependency "Braintree/PaymentFlow", "~> 5.0.0-beta"
+    s.dependency "Braintree/PayPal", "~> 5.0.0-beta"
```

Comment:
Do these dependencies automatically pull in the latest beta version or does this need to be `5.0.0-beta1`?

Reply:
This allows any version of the 5.0 beta. So when we release 5.0.0-beta2, it'll allow either beta 1 or beta 2. If we said `5.0.0-beta1`, it would only allow beta 1.